products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,315 @@
+There is the term, El Clasico, for a match between those two teams.  Let's find all the transfers between
+FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH (m:Team )-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team)
+WHERE (m.name = "FC Barcelona" AND n.name = "Real Madrid") OR (m.name = "Real Madrid" AND n.name = "FC Barcelona")
+MATCH (t)<-[:TRANSFERRED_IN]-(p:Player)
+RETURN m.name as transferred_from_team, p.name as player_name, n.name as transfered_to_team
+```
+
+
+6) FC Barcelona is one of the most valuable football clubs in the world. Players often want to play there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH (m:Team )-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE m.name = "FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as player_names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you. 
+
+```opencypher 
+MATCH (season:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
+MATCH (t)-[:TRANSFERRED_TO]->(team:Team)
+WHERE season.name = "2003/2004"
+WITH DISTINCT player, team
+RETURN team.name as team_name, COUNT(player) AS number_of_players, collect(player.name) as names_of_players
+ORDER BY number_of_players DESC, team_name
+LIMIT 20;
+```
+
+8) In great teams, there are  players who seem to be irreplaceable. When they leave,
+the club board is often struggling to find a proper replacement for them.Let's find out on which positions
+in season 2015/2016 FC Barcelona spent money.
+
+```opencypher 
+MATCH (:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player),
+(s:Season)<-[:HAPPENED_IN]-(t)-[:TRANSFERRED_TO]->(m:Team)
+WHERE t.fee IS NOT NULL AND (s.name IN ["2015/2016", "2016/2017"]) AND m.name = "FC Barcelona"
+RETURN collect(player.name) AS names, player.position AS position, ROUND(SUM(t.fee)) + 'M €' AS money_spent_per_position
+```
+
+9) But what what was the highest transfer amounts FC Barcelona spent per position from 1992/1993 till 2019/2020?
+
+```opencypher 
+MATCH (:Team )-[:TRANSFERRED_FROM ]->(t:Transfer)<-[:TRANSFERRED_IN ]-(player:Player), 
+(t)-[:TRANSFERRED_TO]->(team:Team)
+WHERE t.fee IS NOT NULL AND team.name="FC Barcelona"
+RETURN MAX(t.fee) + 'M €' AS max_money_spent, player.position as player_position
+```
+
+10) Now, let us find who were the most expensive players per positions in our team FC Barcelona. 
+
+```opencypher 
+MATCH
+    (team:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:HAPPENED_IN]->(s:Season)
+WHERE 
+    t.fee is NOT NULL AND 
+    team.name = "FC Barcelona"
+WITH p.position as player_position, max(t.fee) as max_fee
+MATCH
+    (p:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[:TRANSFERRED_TO]->(team:Team)
+WHERE
+    p.position = player_position AND
```

Comment:
It would be good to decide if you will be using 4 spaces or 2 spaces for the query indent. Regardless of which one you choose, just be consistent. I have seen both.

Reply:
:+1:

Reply:
@antoniofilipovic this is not done yet, here we have 4 lines, on lines 233 and other there are 2 lines. Make it consistent.

Reply:
:+1: